### DISCUSSION TITLE
Как получить изображение

### DISCUSSION THREAD

**Discussion by Ykyki4** [ai-forever/gigachain#59] [Q&A]
Пытаюсь получить изображения, используя подобный код

```python
chat = GigaChat(credentials=env('GIGACHAT_CREDS'), verify_ssl_certs=False)

messages = [
    HumanMessage(content='Нарисуй большого слоника')
]

res = chat(messages)
```

В ответ получаю

```
Вот что у меня получилось: <img src="8882p0090sa2010q1482g4g539b2g08vbmej6p0f9s07pna9816q6n2t9rdjmnta814j4njvan0qpmtf14eq28dct8wky001355jy30f0ge0q9y85n47apapan4qmnjmbcc2a123ax5q6qam1x6j80t3a8e2w00tbs62012pa90phc2sg8" fuse="true"/>
```

Ожидаемый результат -  ссылка на изображение

**Comment by kothulit** [comment]
Это идентификатор изображения, его можно использовать для того, что скачать изображение. Для этого нужно сделать get запрос к эндпоинту, например:
'''python
response = requests.get(f'https://gigachat.devices.sberbank.ru/api/v1/files/{img_src}/content', headers=headers, verify=False)
with open('image.jpg', 'wb') as f:
    f.write(response.content)
'''
если не ошибаюсь сейчас идентификатор в формате uuid, например такой:
<img src="ef3772e3-a0d7-4a79-bdf3-eed56c4d9bb0" fuse="true"/>
